internal/pkg/cli/interfaces.go

```diff
@@ -249,10 +249,36 @@ type wsPipelineManifestReader interface {
 	ReadPipelineManifest(path string) (*manifest.Pipeline, error)
 }
 
+type pathDisplayer interface {
+	// DisplayPath takes any path and returns it in a form ready to be displayed to
+	// the user on the command line.
+	//
+	// No guarantees are given on the stability of the path across runs, all that is
+	// guaranteed is that the displayed path is visually pleasing & meaningful for a
+	// user.
+	//
+	// This path should not be stored in configuration files or used in any way except
+	// for being displayed to the user.
+	DisplayPath(path string) (string, error)
+}
+
+type wsPathRelativizer interface {
```

Comment:
nit: Since the `Rel(path) (string, error)` doesn't have a `workspace` specific type in it we can make the interface name a bit more generic
```suggestion
type relPath interface {
```

Reply:
Good catch, I didn't think to rename the interface when I was removing the other method from it!